### PR TITLE
chore(storybook): include knobs for react wrapper leadspace storybook

### PR DESCRIPTION
**Pull request by annawen1**
### Related Ticket(s)

React wrapper: Lead space - Enable "Knobs" section #6781

### Description

Add storybook knobs to the React Wrapper Leadspace stories to allow for QA to test the component. The knobs currently match the `web-components` leadspace stories.

Screen Shot 2021-08-11 at 4 31 10 PM

### Changelog

**New**

- Add storybook knobs for buttons, image, text, and navigation

### CODE REVIEW

File: packages/web-components/src/components/leadspace/__stories__/leadspace.stories.react.tsx [hunk 1]
```diff
@@ -58,63 +118,168 @@ export const Tall = ({ parameters }) => {
...
+            href: text(`Link ${i + 1}`, `https://example.com`),
+            copy: text(`Button ${i + 1}`, `Button ${i + 1}`),
+            renderIcon: iconMap[icon],
+            label: getAriaLabel(icon),
+          };
+        }),
+      }),
+    },
+  },
+};
```
Comment (kennylam) [inline packages/web-components/src/components/leadspace/__stories__/leadspace.stories.react.tsx:146]:
```suggestion
const leadspaceParams = {
  navElements: select('navigation elements (optional)', navigationOptions, navigationOptions[2]),
  title: text('title (title)', 'Heading can go on two lines max'),
  copy: text('copy (copy)', 'Use this area for a short line of copy to support the title'),
  alt: text('Image alt text (alt)', 'Image alt text'),
  buttons: Array.from({
    length: number('Number of buttons', 2, {}),
  }).map((_, i) => {
    const icon = select(`Icon ${i + 1}`, iconOptions, iconOptions['Arrow Right']) ?? 0;
    return {
      href: text(`Link ${i + 1}`, `https://example.com`),
      copy: text(`Button ${i + 1}`, `Button ${i + 1}`),
      renderIcon: iconMap[icon],
      label: getAriaLabel(icon),
    };
  }),
};

export const Tall = context => Default(context);
Tall.story = {
  parameters: {
    knobs: {
      Leadspace: () => ({
        size: LEADSPACE_SIZE.NONE,
        hasImage: false,
        leadspaceParams,
      }),
    },
  },
};
```

**Review by kennylam — COMMENTED** [review]
Thanks for this @annawen1! Just a suggestion maybe we can eliminate some repetition by putting some of the shared knobs in a var?

File: packages/web-components/src/components/leadspace/__stories__/leadspace.stories.react.tsx (hunk 1 above)
Comment (annawen1) [reply packages/web-components/src/components/leadspace/__stories__/leadspace.stories.react.tsx:146]: good point! will do!

Comment (annawen1) [reply packages/web-components/src/components/leadspace/__stories__/leadspace.stories.react.tsx:146]: actually, tried it out and it doesn't behave correctly. It seems to render out the default and if you try to change the knob, the changes don't reflect.

Comment (kennylam) [reply packages/web-components/src/components/leadspace/__stories__/leadspace.stories.react.tsx:146]: Oh, lame. Ok thanks for trying.

**Review by kennylam — APPROVED** [review]
Thanks @annawen1!

**Pull request merged**